docs/articles_en/documentation/openvino_extensibility/ways_to_get_stateful_model.rst

```diff
@@ -0,0 +1,207 @@
+Ways to get stateful models in OpenVINO {#openvino_docs_OV_UG_ways_to_get_stateful_model}
+==============================
+
+## State related Transformations
+
+If the original framework does not have a special API for working with states, after importing the model, OpenVINO representation will not contain Assign/ReadValue layers.
+For example, if the original ONNX model contains RNN operations, IR will contain TensorIterator operations and the values will be obtained only after execution of the whole TensorIterator primitive.
+Intermediate values from each iteration will not be available. To enable you to work with these intermediate values of each iteration and receive them with a low latency after each infer request,
+special LowLatency2 and MakeStateful transformations were introduced.
+
+### How to get TensorIterator/Loop operations from different frameworks via ModelOptimizer.
+**MXNet:** *LSTM, RNN, GRU* original layers are converted to TensorIterator operation, TensorIterator body contains LSTM/RNN/GRU Cell operations.
```

Comment:
Do not refer to frameworks with deprecated support in OV, like MXNet.

Reply:
deleted